### ISSUE TITLE
Question About Bootable USB

### ISSUE THREAD

**Issue by dbyentzen** [intialonso/intialonso.github.io#4]
I've tried to created a bootable USB by using "dd" to copy the iso to a USB but am not able even after several tries. Is the iso best burned to a CD/DVD instead? Is there a recommended method for creating a bootable live image for installation?

Thanks

**Comment by intialonso** [comment]
Well it is based on lubuntu, dd should work. I have used Unetbootin and the
Ubuntu usb creator without problems.

2016-07-23 21:01 GMT-05:00 dbyentzen <notifications@github.com>:

**Comment by dbyentzen** [comment]
I've used both dd and Unetbootin without success. I've created live images of *buntu spins for testing and installation for a while now. I'm sure it's an error on my part somewhere soI'll keep trying.  Your spin is exactly what I've been looking for to install on my laptop. Great work, thanks.

**Comment by intialonso** [comment]
Is the checksum right? What error do you get with dd and the resulting drive?

2016-07-24 8:50 GMT-05:00, dbyentzen <notifications@github.com>:

**Comment by dbyentzen** [comment]
Yes, cksum is correct. I tried two different brands of pendrives and both FAT and ext4 formats but could not get my Dell Latitude E6420 to boot from a USB with live image. I decided to burn to a DVD and boot from it and it worked from a DVD. Go figure, ha. DVD boots great. 
To answer your query specifically, I received no error only my laptop would boot even in legacy mode with boot option set to USB. 
 I've found everything I'd like to use here & UbuntuQt on a laptop is a perfect match to my Netrunner on my tower.  I've no other place to ask you questions about plans so I'll ask here.
Since you are basing off of 16.04, are there plans to continue long term support? How long? I'm currently using an Xubuntu 16.04 spin but I like UbuntuQt much better. If there are plans to extend the project I want to install and use it but if not I'll stay with what I have( I'm hoping there are plans!).

Thanks a lot for your help!

**Comment by intialonso** [comment]
@dbyentzen My plan is to keep generating the iso until Lubuntu switches officialy to LXQt (Im not sure is this is gonna hapen for 16.10 or 17.04) , then I will stop. Being just one more *buntu, the support regarding packages, kernel, drivers, etc; must be the same than regular Ubuntu.

Please be aware that this iso is using PPAs, so you will be running bleeding edge code; no problems in my experience, but you must be prepared if something arise. If you want to make it more stable I suggest to remove the PPAs.

**Comment by dbyentzen** [comment]
OK, thanks for the info. My feedback---I could never get the iso to boot from any USB. I used the exact same method for it that I've used to install *buntus on several different machines( always worked). My error was a "non-bootable" device. I was able to get a live imagine working using a DVD so I tried to install but failed. The first time I tried to install with FDE and recieved a swap file error message( something about telling my to turn swap off). I turned it off on the live session and tried to re-install without FDE. The install appeared to work but hung at "creating user". In the end I was not able to install. I wish I would have made screenshots for you. 

Thanks

**Comment by MttCastelli** [comment]
@dbyentzen do you have an acer aspire one? I achieve to boot only with a linpus iso USB.

regards

**Comment by dbyentzen** [comment]
I am using a Dell Latitude E6420 series lap. I have used the same method( dd the iso to a USB ) and successfully created a bootable live iso for ChaeltOS( a Xubuntu spin) on the same lap. I'm its working for others.

**Comment by micki59** [comment]
Hallo
I tried the iso but there are problems.usb with start media creator only results in Reboot and put a bootable media in.After made an usb with unnetbootin I had 2 entries,normal boot results in also
reboot and put a bootable media in,Uefi boot looks firstly good.I can try the iso,which looks very good
(thanks for the artwork with dark theme) it also is possible to start installing it but after kernel install
and hardware check it stops cannot install bootloader in target,After this the machine(6years old but 64bit and ssd,normally working well)is frozen and did not react to keyboard or mouse.After controll I
saw that there was an uefi entry in partition list so I dont know whats going wrong:do you make a  new  
version or should I wait(Lubuntu announces progress for the change to lxqt)
With best regards                                                                                                     micki @intialonso @ #5

**Comment by intialonso** [comment]
Ey everybody, Lubuntu is officially switching to LXQt! So you just have to
wait some days for the official ISO to come out!

https://lists.ubuntu.com/archives/ubuntu-release/2016-August/003848.html

2016-08-13 16:07 GMT-05:00 micki59 <notifications@github.com>:
